Developer_Manual.rst

```diff
@@ -828,59 +834,59 @@ Added new CPython suites
 
 When adding a test suite, for a new version, proceed like this, of
 course while adapting of course the names. These are the commands used
-for adding CPython311 based on the CPython310 branch.
+for adding CPython313 based on the CPython312 branch.
 
 .. code:: bash
 
    # Switch to a new branch.
-   git checkout CPython310
-   git branch CPython311
-   git checkout CPython311
+   git checkout CPython312
```

Comment:
Actually there is no CPython312 branch yet, this still needs to be completed for 311, so this part of the change is not yet due.

Reply:
I actually checked beforehand, true 😅
Should I revert this file or is there no "danger" that developers try and fail until missing branches have been added?

Reply:
I will be surprised if anybody but me ever executes the said procedure, sadly. So I guess, I can merge like that. As it stands, it's only example commands anyway.